LYNX-179: Documentation for order cancellation mutation (#191)
AdobeDocs/commerce-webapi: src/pages/graphql/schema/orders/mutations/cancel-order.md

---
title: cancelOrder mutation | Commerce Web APIs
---

> 

Reviewer: Add the following lines above the head1 and be sure to include a blank line before the heading.

```
import BetaNote from '/src/_includes/graphql/notes/beta.md'

<BetaNote />
```

# cancelOrder mutation

The `cancelOrder` mutation allows the owner of the order and admin users to cancel an order, passing an order identifier and a cancellation reason.

> Orders can only be cancelled if `order_cancellation_enabled` config is `true` and if they are in status *RECEIVED* or *PENDING*. If any charge has been done to the customer, the order will be refunded.

Reviewer: You should probably add the status when a refund is required.

> Orders can only be cancelled if `order_cancellation_enabled` config is `true` and if they are in status *RECEIVED* or *PENDING*. If any charge has been done to the customer, the order will be refunded.

Author: what do you mean exactly?

> Orders can only be cancelled if `order_cancellation_enabled` config is `true` and if they are in status *RECEIVED* or *PENDING*. If any charge has been done to the customer, the order will be refunded.

Reviewer: Mention the the state of the order when the customer is due a refund.

If the customer has been charged for the order, the order is placed in the X state, and the customer will be issued a refund.

Or, if the order is always placed in the CANCELED state,

The status of the order is set to CANCELED. If the customer has been charged for the order, they will be issued a refund.

The mutation returns an error if the order cannot be cancelled.

## Syntax

`mutation: {cancelOrder(input: CanceOrderInput!) {CancelOrderOutput}}`

## Example usage

> The following example cancels the order with id `99999999` providing a reason for its cancellation.

Reviewer: ```suggestion
The following example cancels order ID `99999999` and includes the reason for its cancellation.
```
